### PR TITLE
completed ticket 6

### PR DESCRIPTION
**Pull request by yazyounas**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -86,3 +87,33 @@ describe("GET", () => {
...
 });
+
+describe("GET", () => {
+  it("should return array of comments when given review_id with following properties", () => {
+    return request(app)
+      .get(`/api/reviews/2/comments`)
+      .expect(200)
+      .then(({ body }) => {
+        const comments = body.comments;
+        expect(comments.length).toBe(3);
```
Comment (pippaamy) [inline __tests__/app.test.js]: this test will fail if any more comments are added when we check the length here

File: __tests__/app.test.js [hunk 2]
```diff
@@ -86,3 +87,33 @@ describe("GET", () => {
...
+  });
+  it("should return comments sorted by created_at in descending order", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeSortedBy("created_at", { descending: true });
+      });
+  });
```
Comment (pippaamy) [inline __tests__/app.test.js:154]: Lets add some error tests here:
check for status 400, invalid ID, e.g. string of "not-an-id"
check for Status 404, non existent ID, e.g. 0 or 9999

File: __tests__/app.test.js [hunk 3]
```diff
@@ -86,3 +87,33 @@ describe("GET", () => {
       });
   });
 });
+
```
Comment (pippaamy) [inline __tests__/app.test.js:89]: can we add a 200 test for  valid ID, but has no comments responds with an empty array of comments

**Review by pippaamy — CHANGES_REQUESTED** [review]
Great work, some changes to make :)

**Review by callumu21 — APPROVED** [review]
Nice job implementing those changes! This should be good to merge now 👍
